src/kata5.reachDestination.js

```diff
@@ -1,3 +1,5 @@
-const reachDestination = (distance, speed) => {};
-
+const reachDestination = (distance, speed) => {
+const arrivalTime = Math.ceil((distance / speed) * 2) / 2;
+return `I should be there in ${arrivalTime} hours.`;
+};
```

Comment:
You could condense this a bit and have it still be perfectly readable:
```
const reachDestination = (distance, speed) =>
  `I should be there in ${Math.ceil((distance / speed) * 2) / 2} hours.`;

```
Also its good practice to leave a clear line between the last function and the export statement at the end of a file